ew/cmd/slimeoid/slimeoidcmds.py

```diff
@@ -576,29 +596,43 @@ async def slimeoidbattle(cmd):
                     accepted = 1
         except:
             accepted = 0
+        
+        if(accepted == 1):
```

Comment:
the idea behind this is to open up slimeoid hustling/swapping, are you sure we want every slimeoid battle to open up the risk of it? 

this isn't a rhetorical question, i don't know this meta and it genuinely sounds like it could liven things up. but i anticipate players resenting that the slimeoids they agree to battle with are essentially a mystery to the other person until after the agreement

i'll throw out the idea that we could use two different versions of the command. !slimeoidbattle and !sl¡meoidbattle (with an upside down !), each of which applies different rules

Reply:
i cant type !sl¡meoidbattle on my keyboard but i agree different commands for different types would be good. I set !slimeoidduel as an alias and then had the command turn on the count-down only if its a duel, !slimeoidbattle works the same as before.